### PR TITLE
additions

### PR DESCRIPTION
**Pull request by Zam6969**
Skinned armors proctect you wastland deadend holidayhat and 1more , added more hospital items to loot for quests, Ethanol Torch and regular torch can now be placed, Gas Traps , Fire Traps Can be placed. Punji Stick Row Can be placed now. added skin enums and a skin loadout for easy spawning of skins.

### CODE REVIEW

**Comment by Zam6969** [comment]
@avcio9  check it pls

File: src/servers/ZoneServer2016/commands/commands.ts [hunk 1]
```diff
@@ -1622,6 +1623,22 @@ export const commands: Array<Command> = [
       server.sendChatText(client, `Build kit given`);
     },
   },
+  /*{
```
Comment (QuentinGruber) [inline src/servers/ZoneServer2016/commands/commands.ts:1626]: ?

**Comment by Zam6969** [comment]
Fixed.

File: src/servers/ZoneServer2016/managers/constructionmanager.ts [hunk 2]
```diff
@@ -336,7 +332,19 @@ export class ConstructionManager {
     rotation: Float32Array,
     parentObjectCharacterId: string,
     BuildingSlot: string
+    
   ) {
+     {
+      if (client.isAdmin) {
+        // Handle admin placement logic here
```
Comment (rxmario) [inline src/servers/ZoneServer2016/managers/constructionmanager.ts]: this can be removed

File: src/servers/ZoneServer2016/data/loadouts.ts [hunk 3]
```diff
@@ -48,6 +49,67 @@ export const characterKitLoadout = [
   { item: Items.HELMET_MOTORCYCLE },
   { item: Items.CONVEYS_BLUE },
 ];
+export const characterskinsloadout = [
```
Comment (rxmario) [inline src/servers/ZoneServer2016/data/loadouts.ts:52]: camel case

**Comment by QuentinGruber** [comment]
And also use `npm run prettier-sources` to fix the prettier issue :cowboy_hat_face:

File: src/servers/ZoneServer2016/commands/commands.ts [hunk 4]
```diff
@@ -224,6 +225,7 @@ export const commands: Array<Command> = [
           );
         }
       }
+      //server.sendCharacterData(client); //Vanish First Person fix Kinda But breaks your inventory  /////// :/ Walk though a door and the camera will be fixed. -Bryan
```
[the remark this answers is not in the document]
Comment (Zam6969) [reply src/servers/ZoneServer2016/commands/commands.ts:228]: well i added it if people want to use it shall i remove it?

**Comment by Zam6969** [comment]
> And also use `npm run prettier-sources` to fix the prettier issue 🤠

DONE :)

File: src/servers/ZoneServer2016/commands/commands.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (QuentinGruber) [reply src/servers/ZoneServer2016/commands/commands.ts:228]: It's a dirty hack that don't really resolve the issue, feels like it's unnecessary

[the remark this answers is not in the document]
Comment (Zam6969) [reply src/servers/ZoneServer2016/commands/commands.ts]: Removed.

**Comment by Zam6969** [comment]
@QuentinGruber Can you check if i did the damage right with the trap? with my last commit

**Comment by Zam6969** [comment]
oh lol i just did the same thing lmao

**Comment by Zam6969** [comment]
@QuentinGruber would you like me to remove the code that allows admins to place anywhere or keep it cause i know it could cause issues if you have untrust worthy mods unless theres a different way to say what type of admin?

**Comment by Zam6969** [comment]
or have a seperate place for mods?

**Review by Z1Meme — APPROVED** [review]

**Comment by Z1Meme** [comment]
@Zam6969 run prettier and i'll merge it "npm run prettier_sources"

**Review by Z1Meme — APPROVED** [review]

**Review by Z1Meme — APPROVED** [review]

**Pull request merged**